lab-configs.yaml

```diff
@@ -10,6 +10,7 @@ labs:
           plan:
             - baseline
             - baseline-fastboot
```

Comment:
I believe it's worth adding `baseline-nfs` since `kselftest` uses nfsroot, to check it's working with regular kernel configs:
```suggestion
            - baseline-nfs
            - baseline-fastboot
```

Reply:
I left that out on purpose because it's an extra test that doesn't add any value, other than sanity checking NFS.  

Since there's no way to check the status of `baseline-nfs` before running `kselftest`, I don't see any extra value here.

Reply:
Well it can be used to verify manually that nfsroot works as expected, when kselftest jobs fail to run.

Reply:
Sure, but that's orthogonal to this patch, and should be up to the lab owner to decide.  Let's not hold this series up for unrelated issues.

Reply:
Sure, this is not what's holding this PR. It just seemed like an obvious thing to me when I started enabling kselftests on more devices.